ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/MinimalState.java

```diff
@@ -300,7 +300,7 @@ public void defaultVMInstallChanged(IVMInstall previous, IVMInstall current) {
 			}
 		});
 	}
-	private static final String PDE_MANIFEST_BUILDER = "org.eclipse.pde.ManifestBuilder"; //$NON-NLS-1$
+	public static final String PDE_MANIFEST_BUILDER = "org.eclipse.pde.ManifestBuilder"; //$NON-NLS-1$
```

Comment:
The class org.eclipse.pde.internal.core.natures.PDE already provides the constant for this named `MANIFEST_BUILDER_ID`. Actually I should have used that here too and probably just oversaw the existing constant.
Created https://github.com/eclipse-pde/eclipse.pde/pull/941 to change that and I suggest you use that too.